Src/Base/AMReX_NonLocalBCImpl.H

```diff
@@ -163,7 +163,7 @@ local_copy_cpu (FabArray<FAB>& mf, int scomp, int ncomp, FabArrayBase::CommMetaD
     auto const& LocTags = *(cmd.m_LocTags);
     int N_locs = LocTags.size();
     if (N_locs == 0) return;
-#ifdef _OPENMP
+#ifdef _OPENMP // Note: intentionally not AMREX_USE_OMP
```

Comment:
Why is `_OPENMP` used here?  I think the only complicated situation we can support is AMReX is built without OpenMP whereas the downstream projects is built with OpenMP.  In that case, I don't think we should enable OpenMP here because it's part of AMReX.  If we do and this template is instantiated in AMReX itself (although currently that's not the case for this template), we might end up with this template is instantiated in two different ways.  That might be a problem.

Reply:
I think that sounds reasonable, yes.

I was thinking about a similar pattern downstream: in this case this is an auxiliary function/loop. That works well with e.g. CUDA being on. Will change to avoid confusion.